### PR TITLE
kill withSafeNavigation

### PR DESCRIPTION
**Pull request by jakob223**
cc @keybase/y2ksquad

### CODE REVIEW

File: shared/chat/inbox/row/build-team/index.tsx [hunk 1]
```diff
@@ -1,23 +1,38 @@
...
-    direction="vertical"
-    gap={Styles.isMobile ? 'tiny' : 'xtiny'}
-    style={Styles.collapseStyles([styles.container, isFloating && styles.floatingContainer])}
-  >
-    <Kb.Button fullWidth={true} label="Create a team" mode="Secondary" onClick={onCreateTeam} />
-    <Kb.Button fullWidth={true} label="Join a team" mode="Secondary" onClick={onJoinTeam} />
-  </Kb.Box2>
-)
+  const isFloating = Container.useSelector(
+    state => ((state.chat2.inboxLayout && state.chat2.inboxLayout.bigTeams) || []).length === 0
```
Comment (chrisnojima) [inline shared/chat/inbox/row/build-team/index.tsx]: `!!state.chat2.inboxLayout?.bigTeams?.length`

File: shared/chat/inbox/container/index.tsx [hunk 2]
```diff
@@ -204,19 +205,18 @@ const Connected = Container.namedConnect(
...
       rows,
       setInboxNumSmallRows: dispatchProps.setInboxNumSmallRows,
       smallTeamsExpanded: stateProps.smallTeamsExpanded,
-      title: 'Chats',
       toggleSmallTeamsExpanded: dispatchProps.toggleSmallTeamsExpanded,
       unreadIndices,
     }
   },
-  'Inbox'
+  'Chats'
```
Comment (chrisnojima) [inline shared/chat/inbox/container/index.tsx]: ?? this can break stories

**Comment by chrisnojima** [comment]
mostly seems fine to me, @songgao should take a look at the fs stuff

File: shared/chat/inbox/container/index.tsx (hunk 2 above)
Comment (jakob223) [reply shared/chat/inbox/container/index.tsx]: I think I had removed this line in order to use `connectDEBUG` and then when I added it back I added the wrong thing

**Pull request merged**